integration_tests/src/payments_althea.rs

```diff
@@ -1,2 +1,125 @@
-/// Runs a five node fixed network map test scenario
-pub fn run_althea_payments_test_scenario() {}
+use std::thread;
+use std::time::Duration;
+
+use crate::five_nodes::five_node_config;
+use crate::setup_utils::database::start_postgres;
+use crate::setup_utils::namespaces::*;
+use crate::setup_utils::rita::thread_spawner;
+use crate::utils::{
+    althea_system_chain_client, althea_system_chain_exit, generate_traffic, get_default_settings,
+    print_althea_balances, query_debts, register_to_exit, send_althea_tokens, test_reach_all,
+    test_routes, NODE_IP,
+};
+use althea_types::ALTHEA_PREFIX;
+use deep_space::Address as AltheaAddress;
+use deep_space::{EthermintPrivateKey, PrivateKey};
+use log::info;
+
+const USDC_TO_WEI_DECIMAL: u64 = 1_000_000_000_000u64;
+
+const ALTHEA_EVM_PRIV_BYTES: &str =
+    "3b23c86080c9abc8870936b2eb17ecb808f5ad3b318018b3e23873013379e4d6";
```

Comment:
it looks like this is just a random testing key, it would be better to have a utility function that generates a random key rather than using a hardcoded one,. A random key is both a better test (may find issues that only some keys have) and prevents things like hardcoding addresses in the test that don't need to be constants.

Reply:
I grab this from setup-validators.sh